src/LoginForm/LoginForm.js

```diff
@@ -1,24 +1,56 @@
 import React, { Component } from 'react';
+import { NavLink } from 'react-router-dom';
 import './LoginForm.css';
+import Areas from '../Areas/Areas';
+import map from '../assets/map.png';
 
 class LoginForm extends Component {
   constructor() {
     super();
+    this.state = {
+      name: '',
+      email: '',
+      type: ''
+    }
+  }
+
+  handleChange = (event) => {
+    let key = event.target.id || event.currentTarget.value;
+    this.setState({ [key]: event.target.value })
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    if (this.state.name === '' || this.state.email === '' || this.state.type === '') {
+      event.preventDefault();
+      document.querySelector('.error').classList.remove('hidden');
+    } else {
+      this.props.history.push('/areas')
+    }
   }
 
   render() {
     return (
-      <form className='login-form'>
-        <input type='text' id='name-input' placeholder='NAME'/>
-        <input type='text' id='email-input' placeholder='EMAIL'/>
-        <select>
-          <option value="" disabled selected>TYPE OF TRAVEL </option>
-          <option value='business'>BUSINESS</option>
-          <option value='vacation'>VACATION</option>
-          <option value='other'>OTHER</option>
-        </select>
-        <button id='submit-button'>SUBMIT</button>
-      </form>
+      <div className='login-screen'>
```

Comment:
This is a cleaner change than what we had previously.  I like the inclusion of  the header/logo in the form instead of as a separate component.